apps/api/src/index.ts

```diff
@@ -126,6 +126,6 @@ initAPI();
 
 function openInBrowser() {
     if (process.env.NODE_ENV === "development") {
-        exec(`start http://localhost:${settings.port}`);
+        exec(`start http://localhost:${settings?.port}`);
     }
 }
```

Comment:
```suggestion
```
Can be removed, no longer used.

Reply:
the ?. does not really mean anything, its just to make typescript happy. it has to be there because the settings variable can be null